src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/integration/ProcessUnknownPatientsIntTest.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.integration
+
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.mockito.ArgumentMatchers.anyList
+import org.mockito.kotlin.whenever
+import org.springframework.test.context.ActiveProfiles
+
+@ActiveProfiles("test")
+class ProcessUnknownPatientsIntTest : IntegrationTestBase() {
+
+  @Nested
+  inner class Security {
```

Comment:
Started with tests to secure the endpoint. More integration tests to come for the functionality.